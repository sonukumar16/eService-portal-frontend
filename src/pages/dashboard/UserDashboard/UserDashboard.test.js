import { shallow } from "enzyme";
import { cleanup } from "@testing-library/react";
import renderer from 'react-test-renderer';
import { Typography } from '@material-ui/core';
import UserDashboard from './';


describe('UserDashboard test cases', () => {

    const wrapper = shallow(<UserDashboard />);

    afterEach(cleanup);

    it('should match snapshot', () => {
        const tree = renderer.create(wrapper).toJSON();
        expect(tree).toMatchSnapshot();
    });

    it('should render welcome message', () => {
        expect(wrapper.find(Typography).text()).toEqual("Welcome to user dashboard");
    });

});
