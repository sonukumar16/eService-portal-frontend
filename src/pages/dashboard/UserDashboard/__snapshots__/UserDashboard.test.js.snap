// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserDashboard test cases should match snapshot 1`] = `
<main
  className="MuiContainer-root makeStyles-container-1 MuiContainer-maxWidthLg"
>
  <h1
    className="MuiTypography-root MuiTypography-h5"
  >
    Welcome to user dashboard
  </h1>
</main>
`;
